test/e2e/common/pods.go

```diff
@@ -771,8 +771,13 @@ var _ = framework.KubeDescribe("Pods", func() {
 		}
 	})
 
+	/*
+		Release : v1.15
+		Testname: Ensure readiness gates are respected
+		Description: Create a pod with multiple readiness gates and ensure the pod is unReady until all gates are ready
```

Comment:
Rewrite this in more spec-y language:

Create a pod with multiple readiness gates. The pod MUST become ready only after all gates are ready.

Reply:
RFC2119 style with CAPS or not :smile: ?
```
The pod MUST become ready ONLY after all gates are ready.
```

Reply:
The description has been updated to be to the point.